### ISSUE TITLE
pquery/execute are broken due to non-existent pgsql_util:decode_row/2 call

### ISSUE THREAD

**Issue by drdaeman** [processone/pgsql#3]
Just wanted to write a small site-specific ejabberd module that uses PostgreSQL directly (without `ejabberd_odbc` etc) and stumbled upon an issue.

The `pgsql_util` module only exports `decode_row/3`, but a call to non-existent `decode_row/2` is done in few places at `pgsql_proto` (lines 414 and 495 as of a36f2d1).

Effectively, this means `pgsql:execute` and related functions are unusable for any queries returning rowsets, as any such call would result in `'function not exported'` exception.

**Comment by badlop** [comment]
That problem is confirmed by Dialyzer:
```
pgsql_proto.erl:414, Dialyzer, Priority: Normal
Call to missing or unexported function pgsql_util:decode_row/2

pgsql_proto.erl:495, Dialyzer, Priority: Normal
Call to missing or unexported function pgsql_util:decode_row/2
```

**Comment by zinid** [comment]
Fixed in e72c03c60bfcb56bbb5d259342021d9cb3581dac
